js/researches/spanish/firstWordExceptions.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns an array with exceptions for the sentence beginning researcher.
+ * @returns {Array} The array filled with exceptions.
+ */
+module.exports = function() {
+	return [ "el", "los", "la", "las", "un", "una", "unos", "unas", "unos", "dos", "tres", "cuatro", "cinco", "seis",
```

Comment:
I would categorize these words and add a comment before all groups description what the words are so it is clear why they are exceptions.